knnvector/test_procedures/default.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "no-train-test",
+    "default": true,
```

Comment:
To conform with other official workloads and since this is the default test procedure, it'd be helpful to add a "description" field with a blurb about what this test procedure does.

Reply:
Ack